src/main/java/com/bookstore/dto/user/UserLoginResponseDto.java

```diff
@@ -0,0 +1,10 @@
+package com.bookstore.dto.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
```

Comment:
```suggestion
```
`@Data`
Generates getters for all fields, a useful toString method, and hashCode and equals implementations that check all non-transient fields. Will also generate setters for all non-final fields, as well as a constructor.
Equivalent to `@Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode.`

Reply:
Since the token field is not final I decided to use @AllArgsConstructor because @Data only generates the @RequiredArgsConstructor

Reply:
But why `token` field is not final?)
You don't need to create an instance of `UserLoginResponseDto` without token, so this field is mandatory
You can even make it `record` not a class (records have final fields only)